skbio/stats/composition.py

```diff
@@ -608,10 +611,269 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0-dev")
+def ancom(table, groups,
+          alpha=0.05,
+          tau=0.02,
+          theta=0.1,
+          multiple_comparisons_correction=None,
+          significance_test=scipy.stats.ttest_ind):
```

Comment:
Passing a function reference renders oddly with Sphinx:

![screen shot 2015-10-22 at 2 14 16 pm](https://cloud.githubusercontent.com/assets/1847232/10678450/4b4339de-78c7-11e5-92d3-23cff6f502de.png)

Can you make this default to `None` and internally it will use the test function?